environment.md

```diff
@@ -59,23 +75,16 @@ The priority of the transports are Direct file IO, SSH2, FTP PHP Extension, FTP
 
 ## Database
 
-WordPress stores content, configuration, and other information inside of a database. The database for a WordPress website is where all of the WordPress website's user-defined data is stored. WordPress is primarily designed to use a MySQL or MySQL-related database server; WordPress officially only supports MySQL or MariaDB, a drop-in replacement for MySQL.
-
-### MySQL
-
-MySQL is a widely used relational database server. MySQL comes in both open-source and commercial distributions. Either distribution should work with MySQL. The commercial distribution has additional features not found in the open-source distribution; however, WordPress does not require or use the additional features. It is designed to run on either the open-source or the commercial distribution.
-
-### MariaDB
-
-MariaDB is a drop-in replacement for MySQL supported by WordPress. It's a fork of the open-source distribution of MySQL. MariaDB was originally created to maintain a more open-source version of MySQL, but it has grown into its own relational database server alternative to MySQL with features and changes not found in MySQL. Despite its differences, MariaDB is still a fully compatible replacement for MySQL and can generally seamlessly replace MySQL.
+For data storage, WordPress uses systems based on MySQL distributions, such as MySQL, MariaDB or Percona. All three systems are compatible with WordPress, so at this moment any of the three can be used for its operation.
 
-### Percona
+The use of these versions is recommended, both for performance and security reasons, although previous versions usually work without problems:
 
-Percona server is a drop-in replacement for MySQL, focused on performance. Although it's a drop-in replacement for MySQL, WordPress does not officially support Percona.
+* [MariaDB](https://mariadb.org/) 10.4.x / 10.3.x / 10.2.x
```

Comment:
Where did the recommended versions come from, in this case?
For MariaDB, "MariaDB version 10.1 or greater." is listed here: https://wordpress.org/about/requirements/

To make this easier to keep updated, I'm wondering if it might make sense to avoid noting specifics, but with a note to keep up to date with security patches for the version/distro that is being used.

Reply:
Yeah! I think we should add MariaDB 10.1 and MariaDB 5.5, because there are stable and maintenance versions also. Also, yeah, 5.5 is slooow. :P

Reply:
👍